influxdb3/src/commands/create.rs

```diff
@@ -33,7 +35,7 @@ pub fn command(config: Config) -> Result<(), Box<dyn Error>> {
                 HTTP requests require the following header: \"Authorization: Bearer {token}\"\n\
                 This will grant you access to every HTTP endpoint or deny it otherwise
             ",
-                hashed = hex::encode(&Sha512::digest(&token)[..])
+                hashed = B64.encode(&Sha512::digest(&token)[..])
```

Comment:
I think based on my last review comment, this should drop the `B64.encode()`. I missed that we were doing `hex::encode()` for this before, but internally we only need worry about the SHA512 (which is already in hex).